app/src/main/java/nl/tudelft/pixelperfect/client/ConnectTask.java

```diff
@@ -0,0 +1,35 @@
+package nl.tudelft.pixelperfect.client;
+
+import android.os.AsyncTask;
+
+import com.jme3.network.Client;
+import com.jme3.network.Network;
+import com.jme3.network.serializing.Serializer;
+
+import java.io.IOException;
+
+/**
+ * Created by jesse on 10-5-2016.
+ */
+public class ConnectTask extends AsyncTask<String, Void, Client> {
+
+    private Exception exception;
+    public ConnectResponse delegate;
+
+    protected Client doInBackground(String... ip) {
```

Comment:
apparently this as a [varargs](http://stackoverflow.com/questions/3158730/java-3-dots-in-parameters), cool.